src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,10 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException(
+                    String.format("Expected non-negative number, got ?", number));
```

Comment:
did you test, if it shows number correctly? 
you may wanna use %d instead of ?

Reply:
yes, all tests are passed. Done